docs/users-guide/models.md

```diff
@@ -110,13 +110,34 @@ Just like with a question, admins can verify models. Verifying a model will give
 
 _Currently available for PostgreSQL, MySQL, and Redshift_.
 
-You can enable caching for models so that loading questions based on those models is significantly faster. Instead of running the query that creates the model fresh each time, Metabase will periodically run the query and store the results in a table in your data warehouse, so that whenever someone loads the model (or a question based on that model), Metabase will simply fetch the saved results from the table in your data warehouse.
-
-To store cached results, Metabase creates a new schema in your data warehouse, and creates new tables in that schema to store the results of the queries that power the models. In order for Metabase to cache models in your database, the connection credentials you use to connect Metabase to your data warehouse need to be able to create new schemas in your data warehouse. So, for example, if you're connected to a PostgreSQL database, the Postgres user account that you use to connect Metabase to that Postgres database must have privileges to create new schemas in that Postgres database.
+You can enable caching for models so that loading questions based on those models is significantly faster. Instead of running the query that creates the model fresh each time, Metabase will periodically run the query and store the results in a table in your data warehouse, so that whenever someone loads the model (or a question based on that model), Metabase will simply fetch the saved results from that table in your data warehouse.
 
 ### Enable model caching for a database
 
-To enable model caching, click on the **gear** icon in the upper right and select **Admin settings** > **Settings** > **Caching**.
+There are two steps to enabling model caching for your database.
+
+1. [Create a schema in your database to store cached models](#1-create-a-schema-in-your-database-to-store-cached-models)
+2. [Enable model caching in your Metabase](#2-enable-model-caching-in-your-metabase)
+
+#### 1. Create a schema in your database to store cached models
+
+To store cached results, Metabase creates a new schema in your data warehouse, and creates new tables in that schema to store the results of the queries that power your models.
+
+To create the schema in your database, click on the **gear** icon in the upper right and select **Admin settings** > **Databases**, then select the relevant database connection. On the right, click on **Turn model caching on**.
```

Comment:
This might be a bit much. Metabase will attempt to create the schema on its own. I think for the majority of people it will just work. For larger shops with more locked down schemas, they will most likely need to create the schema themselves and ensure the db user has permissions. But to me this for more knowledgeable users.

We could make it clear that Metabase will check if the schema exists, or attempt to create it otherwise. If the connection's credentials lack the necessary permissions to create the schema, you will need to create it.